### ISSUE TITLE
Project Maintainer

### ISSUE THREAD

**Issue by PhoenixPeca** [lrsjng/h5ai#692]
Hi @lrsjng ,

Looks like this project has been dormant for quite some time already. I'd like to apply as a project maintainer if you need.

Thanks!

**Comment by shirshak55** [comment]
@PhoenixPeca  

How about we create a new organization and maintain with community support.

And I would like to use rust instead of PHP on new version as this project requires abit performance or we can discuss about it too.:)

Can we make plans etc?


Thanks

**Comment by badda71** [comment]
Sounds like a plan :-) Please take over - I have a couple of PRs to donate to the project

**Comment by hyperclock** [comment]
I think sticking to PHP and if there are a few that want RUST then fork it into a next generation. That's the way I'd go and see if the 2 groups may be able to help each  other. That's the beauty in Opensource, CHOICE and Cooperation!

**Comment by shirshak55** [comment]
@hyperclock yeah only few part in rust which needs more optimization :) . Yea its not good to rewrite whole thing :)

**Comment by PhoenixPeca** [comment]
I have quite a few things in my mind for this project.. I should be forking this project soon.. But for now, I'm pre-occupied with something else. Maybe this can wait guys ;)

**Comment by shirshak55** [comment]
hmm he seems to be back don't know if we need to start from fresh :)

**Comment by PhoenixPeca** [comment]
Hi everyone! since this project may no longer be maintained in the future. I will take over its development soon in a new organization. Is anyone else interested on this?

**Comment by badda71** [comment]
I‘m in

**Comment by shirshak55** [comment]
I'm in

**Comment by djpbessems** [comment]
I'm interested; probs should make a dedicated account for it? So we'll have `github.com/h5ai/h5ai`?

**Comment by shirshak55** [comment]
We can create organiization but user h5ai is already taken so cannot use :(

**Comment by PhoenixPeca** [comment]
Nice, I will create a fork of this project in an existing organization that I [manage](https://github.com/azetrix). I will recreate the project within this week and merge some of the cool changes that I've found in the PRs as well as my own additions. I'll just let you know once it's ready, so that I can add all of you as contributors.

**Comment by PhoenixPeca** [comment]
I once added a functionality to login protect files. You can check it at https://github.com/RAKwirelessDev/stash-finix/

Live demo:

https://stash.rakwireless.com/

username: aspaciop@rakwireless.com
password: admin

**Comment by djpbessems** [comment]
>We can create organiization but user h5ai is already taken so cannot use :(

That account is inactive and has an unused/unedited fork of a random Google-repos to appear in use. 
https://help.github.com/en/articles/name-squatting-policy

**Comment by hartmark** [comment]
Is there any effort to continue this project?

**Comment by hartmark** [comment]
Any news?

**Comment by badda71** [comment]
Why don't you jump in, create a fork and maintain the project from now on?

**Comment by hartmark** [comment]
I lack the PHP development skills to do any good 😒

I can create a fork and incorporate the outstanding PRs if no-one else steps up

**Comment by Voidi** [comment]
As far a i can see this project is not dead, but is it still active developed or are there forks which do active development?

**Comment by hartmark** [comment]
No disrespect, but checking the issues-list and PR-queue to the commit-log it seems the project is quite nearly dead. Most issues and PRs goes without any comments.

I guess the original maintainer just keeps this on life support when his time permits.

I lack sadly the php skill and also don't have much extra time on my hands.

**Comment by arafatamim** [comment]
Yeah, the author seems to be quite inactive on this and I haven't seen him commenting on any of the long backlog of issues. I think this is dead too.
